tictactoe.py

```diff
@@ -13,17 +12,35 @@ def draw(self):
         for row in self.grid:
            print(y, row)
            y += 1
- 
-    def makemove(self, player, row, col):
+
+    def make_move(self, player, row, col):
         x = int(row)
         y = int(col)
         self.grid[x][y] = playerLetter(player)
- 
+
+    def check_col(self, x, player):
```

Comment:
These methods should be part of the Board class. Moved them into the class. They should also be private. Learn about private vs public methods and how they are implemented in python and update these (python doesn't really have private, just a naming convention).